OpenSimRT/RealTime/include/RealTimeAnalysis.h

```diff
@@ -0,0 +1,234 @@
+/**
+ * -----------------------------------------------------------------------------
+ * Copyright 2019-2021 OpenSimRT developers.
+ *
+ * This file is part of OpenSimRT.
+ *
+ * OpenSimRT is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * OpenSimRT is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+ * A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * OpenSimRT. If not, see <https://www.gnu.org/licenses/>.
+ * -----------------------------------------------------------------------------
+ *
+ * @file RealTimeAnalysis.h
+ *
+ * @brief This file contains a facade class as a convinient interface for
+ * performing musculoskeletal analysis in mocap scenarios.
+ *
+ * @author Dimitar Stanev <jimstanev@gmail.com>
+ * contribution: Filip Konstantinos <filip.k@ece.upatras.gr>
+ */
+#pragma once
+
+#include "CircularBuffer.h"
+#include "OpenSimUtils.h"
+#include "SignalProcessing.h"
+#include "InverseDynamics.h"
+#include "InverseKinematics.h"
+#include "MuscleOptimization.h"
+#include "JointReaction.h"
+#include "RealTimeAnalysis.h"
+#include "internal/RealTimeExports.h"
+
```

Comment:
I do not like that clang-formatter leaves space in the include. Do you think we should configure it to keep all includes together?

Reply:
I'll check. But personally I don't mind it. It (usually) separates the `"project-headers.h"` from the `<library-headers>`

Reply:
It is good that it sorts things, but let see if it avoids adding new lines.

Reply:
It's an option of the clangd formatter. `IncludeBlocks:Merge`. https://clang.llvm.org/docs/ClangFormatStyleOptions.html